apps/spruce/src/pages/projectAndRepoSettings/shared/tabs/GeneralTab/getFormSchema.tsx

```diff
@@ -138,6 +138,16 @@ export const getFormSchema = (
               true,
             ),
           },
+          debugSpawnHostsDisabled: {
```

Comment:
I think it would make sense for this to go after repotracker, since that's probably a more important global section